src/algo_loc.rs

```diff
@@ -85,28 +94,29 @@ pub fn perform_for_single_line(
                 let count = res.get(&single_struct.author_full_name).unwrap() + 1;
                 res.insert(single_struct.author_full_name, count);
                 continue;
+            } else {
+                res.insert(single_struct.author_full_name, 0);
             }
-            res.insert(single_struct.author_full_name, 0);
         } else {
             for each_file in single_struct.contextual_file_paths {
                 if res.contains_key(&each_file) {
                     let count = res.get(&each_file).unwrap() + 1;
                     res.insert(each_file, count);
-                    continue;
+                } else {
+                    res.insert(each_file, 0);
                 }
-                res.insert(each_file, 0);
             }
         }
     }
     db_obj.store();
     let mut res_string: String = String::new();
     for key in res.keys() {
+        if current_output.contains(key) {
+            continue;
+        }
```

Comment:
Passing the output so far and checking if it is already present was the fix here. (since we now process per line, this was missed in the last PR - perks of having no tests 😢 - gotta prioritise it after DB sharding)